mantidproject/mantid
# Update scale and shift used in reduction

**Description of work.**

This updates the scale and shift values after a SANS reduction so the actual values used are displayed in the GUI. This also produces a rescaled HAB workspace in the case of a merged reduction. 

* Updated view to make values changeable and set default values.
* output and update values from algorithm
* scale seperate HAB appropriately

**Report to:** Sarah 

**To test:**

 Code review
 * The data for this PR was too large to upload here if you want to test it email me and I'll send it to you.
 * Make sure the data is added to your directories path and then load USER_SANS2D_154E_2p4_4m_M3_Xpress_8mm_SampleChanger_REAR.txt as the user file and batch_file.csv as the batch file.
 * Go to Settings->General and change the reduction mode to merged and tick the two use fit boxes in the Merged settings box.
 * Go back to the run tab and click process. 
 * Hover over the row and check that updates scale and shift values are displayed.
 * Check that plotting the merge, front and rear produce results which look like the below.

![image](https://user-images.githubusercontent.com/32058966/48061813-093c9f00-e1b8-11e8-86b0-40a77d97ffbc.png)




Fixes #23309 




---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
